### ISSUE TITLE
Testing Accuracy?

### ISSUE THREAD

**Issue by BrianSantoso** [mitiku1/Emopy-CapsNet#3]
What testing accuracy were you able to achieve with the CapsNet on each dataset? Also, did you do any data preprocessing?

**Comment by BrianSantoso** [comment]
I tried to train on the fer2013 dataset and the test accuracy converged to 13.35%

**Comment by TTTJJJWWW** [comment]
when i run this, the loss = nan
what is the problem?

**Comment by TTTJJJWWW** [comment]
@BrianSantoso maybe this why the test accuracy converged to 13.35%
